src/LIBMONGOC_VERSION_CURRENT

```diff
@@ -1 +1 @@
-1.16.0-20191029+gitce14000d77
+1.11.1-20191113+gitbef4cf349d
```

Comment:
See https://github.com/mongodb/mongo-php-driver/pull/1060#discussion_r345918860.

Reply:
Will rebase once #1060 is merged, holding off on merging before then.